src/DeviceListener.ts

```diff
@@ -282,9 +284,9 @@ export default class DeviceListener {
                 }
             }
         }
-
+        const showDesktopNotificationPrompt = async () => SettingsStore.getValue(UIFeature.ShowSecurityKeyBackupPrompt);
```

Comment:
Should this local variable be called showSecurityKeyBackupPrompt?  Why is the async() => part of the code required?

Reply:
async waits so that if value is loading then it will not execute condition.

Reply:
Other places in the code just do a standard synchronous call, which is what we want anyway

Reply:
No worries too easy, I have removed "async".